### PR TITLE
chore: use MIT license for code and CC-BY license for metadata/docs

### PR DESCRIPTION
**Pull request by dwsilk**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,395 +1,21 @@
...
-to any of its public licenses or any other arrangements,
-understandings, or agreements concerning use of licensed material. For
-the avoidance of doubt, this paragraph does not form part of the public
-licenses.
-
-Creative Commons may be contacted at creativecommons.org.
\ No newline at end of file
+MIT License
+
+Copyright (c) 2023 Land Information New Zealand
```
Comment (blacha) [inline LICENSE:3]: Basemaps has this as 

> Crown copyright (c), Toitū Te Whenua Land Information New Zealand on behalf of the New Zealand Government.

Comment (dwsilk) [reply LICENSE:3]: I knew we ditched the year somewhere, couldn't find it

Comment (blacha) [reply LICENSE]: I was also referencing the "crown copyright" and "Toitū Te Whenua" parts

Comment (dwsilk) [reply LICENSE]: Fixed

**Review by blacha — APPROVED** [review]

**Pull request merged**